lib/plugins/aws/deployFunction/index.js

```diff
@@ -66,9 +66,15 @@ class AwsDeployFunction {
   }
 
   deployFunction() {
-    const artifactFileName = this.provider.naming
-      .getFunctionArtifactName(this.options.function);
-    const artifactFilePath = path.join(this.packagePath, artifactFileName);
+    const functionObject = this.serverless.service.getFunction(this.options.function);
+    const funcPackageConfig = functionObject.package || {};
+    let artifactFilePath = funcPackageConfig.artifact;
```

Comment:
The error showed up in travis as well.
```
Failed to parse file: /home/travis/build/serverless/serverless/lib/plugins/aws/invokeLocal/fixture/handlerWithError.js
Transformation error; return original code
{ [Error: Line 1: Unexpected identifier]
  lineNumber: 1,
  description: 'Unexpected identifier',
  index: 20 }
```
Not sure why the build exited with 0. 🤷‍♀️